https://issues.redhat.com/browse/ACM-11428 z-stream ACM 2.10.3 (#6433)
open-cluster-management/rhacm-docs: release_notes/errata.adoc

See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.

== Errata 2.10.3

* Adds a missing log message to report an error when the `ConfigurationPolicy` controller failed to complete a dry-run update when comparing the policy to the object on the cluster. (link:https://issues.redhat.com/browse/ACM-10612[ACM-10612])

* Fixes an issue when a policy was recreated quickly after deletion, it sometimes led to the compliance status not being populated on the managed cluster. (link:https://issues.redhat.com/browse/ACM-10664[ACM-10664])

* Fixes an issue that caused some unnecessary logs in the `governance-policy-framework` pod to be displayed with the default log verbosity settings. (link:https://issues.redhat.com/browse/ACM-10693[ACM-10693])

* Fixes an issue that required a restart of the `governance-policy-framework` pod when the Gatekeeper operator was installed or uninstalled. The {product-title} integration with Gatekeeper is now enabled or disabled without restarting the pod. (link:https://issues.redhat.com/browse/ACM-10966[ACM-10966])

> * Fixes a bug where a `ConfigurationPolicy` with the `MustOnlyHave` compliance type did not factor root-level keys in the object on the cluster when comparing it to the policy definition. (link:https://issues.redhat.com/browse/ACM-10877[ACM-10877])

Reviewer: ```suggestion
* Fixes a bug where a `ConfigurationPolicy` resource with the `MustOnlyHave` compliance type did not factor root-level keys in the object on the cluster when comparing it to the policy definition. (link:https://issues.redhat.com/browse/ACM-10877[ACM-10877])
```

> * Fixes an issue in the Policy Generator where some placement overrides that exist outside of `policyDefaults` did not correctly override the default. (link:https://issues.redhat.com/browse/ACM-11075[ACM-11075])

Reviewer: ```suggestion
* Fixes an issue in the Policy Generator where some placement overrides that exist outside of the `policyDefaults` parameter section did not correctly override the default. (link:https://issues.redhat.com/browse/ACM-11075[ACM-11075])
```

> * Fixes an issue that prevented the {product-title-short} `gitopsCluster` controller from automatically generating the managed cluster secret for the Argo CD push model when an application addon service account named `application-manager` was provisioned in {ocp} 4.15 by some cloud providers. (link:https://issues.redhat.com/browse/ACM-11149[ACM-11149])

Reviewer: ```suggestion
* Fixes an issue that prevented the {product-title-short} `gitopsCluster` controller from automatically generating the managed cluster secret for the Argo CD push model when an application add-on service account named `application-manager` was provisioned in {ocp} 4.15 by some cloud providers. (link:https://issues.redhat.com/browse/ACM-11149[ACM-11149])
```
